Resources/Private/FusionModule/Components/Links.List.fusion

```diff
@@ -27,12 +25,9 @@ prototype(CodeQ.LinkChecker:Component.Links.List) < prototype(Neos.Fusion:Compon
 prototype(CodeQ.LinkChecker:Component.Links.List.Link) < prototype(Neos.Fusion:Component) {
     link = '{}'
 
-    markAsDoneAction = Neos.Fusion:UriBuilder {
-        action = 'markAsDone'
-    }
-    ignoreAction = Neos.Fusion:UriBuilder {
-        action = 'ignore'
-    }
+    // TODO: link to content module to edit the node
+
+    // TODO: create target link only if protocol is http/https but not for nodes
```

Comment:
@rolandschuetz Here I need some help how to do that

Reply:
Try: 

```
target = Neos.Fusion:Case {
    isNode {
        condition = ${String.startsWith('node://', link.target)}
        renderer = Neos.Fusion:UriBuilder {
            package = 'Neos.Neos.Ui'
            controller = 'Backend'
            action = 'index'
            // Please replace with the correct path to node
            arguments.node = ${link.node}
        }
    }
    
    isResource {
        // tbd
    }
    
    isExternal {
        condition = ${String.startsWith('http', link.target)}
        renderer = ${link.target}
    }
}
```